lib/ansible_tower_client/base_models/workflow_job_node.rb

```diff
@@ -12,7 +12,13 @@ def job
     def job?
       return false if !respond_to?(:job_id) || job_id.nil?
 
-      related.job.match?(/jobs/)
+      related.job.include?("/jobs/")
```

Comment:
This fixes the issue where a nested workflow job would return that `job?` was true, because `"/api/v2/workflow_jobs/1234".match?(/jobs/)` is `true`, to fix this it seems safer to look for `"/jobs/"` instead of just `"jobs"`